be/src/runtime/descriptors.cpp

```diff
@@ -399,7 +399,10 @@ int RowDescriptor::get_row_size() const {
 }
 
 int RowDescriptor::get_tuple_idx(TupleId id) const {
-    CHECK_LT(id, _tuple_idx_map.size()) << "RowDescriptor: " << debug_string();
+    DCHECK_LT(id, _tuple_idx_map.size()) << "RowDescriptor: " << debug_string();
```

Comment:
DCHECK also would be core dump in DEBUG MODE, we should comment DCHECK to make sqlsmith run further.